seth/contract_store_test.go

```diff
@@ -10,40 +10,102 @@ import (
 )
 
 func TestSmokeContractABIStore(t *testing.T) {
-
 	type test struct {
-		name    string
-		abiPath string
-		err     string
+		name              string
+		abiPath           string
+		gethWrappersPaths []string
+		err               string
+		expectedABICount  int
 	}
 
 	tests := []test{
 		{
-			name:    "can load the ABI",
-			abiPath: "./contracts/abi",
+			name:             "can load the ABI only from ABI files",
+			abiPath:          "./contracts/abi",
+			expectedABICount: 5,
+		},
+		{
+			name:              "can load the ABI from ABI files and from gethwrappers",
+			abiPath:           "./contracts/abi",
+			gethWrappersPaths: []string{"./contracts/bind"},
+			expectedABICount:  10,
+		},
+		{
+			name:              "can load the ABI only from gethwrappers",
+			gethWrappersPaths: []string{"./contracts/bind"},
+			expectedABICount:  5,
+		},
+		{
+			name:              "can load the ABI from 2 gethwrappers folders",
+			gethWrappersPaths: []string{"./contracts/bind", "./contracts/bind2"},
+			expectedABICount:  6,
 		},
 		{
 			name:    "can't open the ABI path",
 			abiPath: "dasdsadd",
 			err:     "open dasdsadd: no such file or directory",
 		},
+		{
+			name:              "can't open the gethwrappers path",
+			gethWrappersPaths: []string{"dasdsadd"},
+			err:               "lstat dasdsadd: no such file or directory",
+		},
+		{
+			name:              "correct and broken gethwrappers path",
+			gethWrappersPaths: []string{"./contracts/emptyContractDir", "dasdsadd"},
+			err:               "lstat dasdsadd: no such file or directory",
+		},
+		{
+			name:              "broken and correct gethwrappers path",
+			gethWrappersPaths: []string{"dasdsadd", "./contracts/emptyContractDir"},
+			err:               "lstat dasdsadd: no such file or directory",
+		},
 		{
 			name:    "empty ABI dir",
 			abiPath: "./contracts/emptyContractDir",
 		},
 		{
-			name:    "invalid ABI inside dir",
+			name:              "empty gethwrappers dir",
+			gethWrappersPaths: []string{"./contracts/emptyContractDir"},
```

Comment:
Maybe we should throw an error if the dir is empty? Misconfiguration can happen but I don't want to run without bindings, I usually have them